README.md

```diff
@@ -13,12 +13,29 @@ If you have suggestions or problems using the extension, please [submit a bug or
 
 If you have completed the above steps, the "welcome" page will open indicating successful installation of the extension.
 
+### Build from github
+
+Dependencies: openssl, npm.
+
+1. Clone the tree.
+2. npm install grunt-crx time-grunt grunt grunt-cli
+3. Add ./node_modules/grunt-cli/bin to PATH. (path+= in zsh).
+4. openssl genrsa -out key.pem
+5. grunt
```

Comment:
This can be simplified by adding `grunt-cli` as a dev dependency (`npm install -D grunt-cli`) and a couple of `package.json` scripts:

```
{
  "name": "thegreatsuspender",
  "version": "0.0.0",
  "description": "A chrome extension for suspending all tabs to free up memory",
  "main": "",
  "scripts": {
    "build": "grunt",
    "generate-key": "openssl genrsa -out key.pem",
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "repository": {
    "type": "git",
    "url": "git://github.com/deanoemcke/thegreatsuspender.git"
  },
  "keywords": [
    "chrome",
    "extension",
    "addon",
    "memory",
    "suspend",
    "tab"
  ],
  "author": "deanoemcke",
  "license": "GPLv2",
  "bugs": {
    "url": "https://github.com/deanoemcke/thegreatsuspender/issues"
  },
  "devDependencies": {
    "grunt": "~0.4.5",
    "grunt-cli": "^1.2.0",
    "grunt-crx": "~1.0.3",
    "time-grunt": "~1.2.1"
  }
}
```

Then the instructions can just be:

Clone the repository and run these commands:
```
npm install
npm run generate-key
npm run build
```

----

npm scripts are run with `./node_modules/.bin` in their path... so for people who don't always keep `./node_modules/.bin` in their path, this does the trick. Also, `npm run build` has become a pretty standard script for projects with front-end code.